dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/resources/de/digitalcollections/cudami/server/backend/impl/database/migration/V10.00.00__DDL_Table_for_Licenses.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS licenses (
+  uuid UUID NOT NULL PRIMARY KEY UNIQUE,
```

Comment:
```suggestion
  uuid UUID NOT NULL PRIMARY KEY,
```
Der ist doch schon `unique`

Reply:
ist jetzt korrigiert